lib/importer/nunjucks/importer/macros/table_view.njk

```diff
@@ -4,10 +4,10 @@
     {% set moreRowsCount = data.extraRecordCount %}
     {% set headers = importerHeaderRowDisplay(session, headerMode) %}
 
-    {{ tableView(headers, rows, caption, hideHeader, moreRowsCount) }}
+    {{ tableView(headers, rows, caption, moreRowsCount, 0, false) }}
 {% endmacro %}
 
-{% macro tableView(headers, rows, caption=false, hideHeader=false, rowsBelow=0, rowNumbers=false) %}
```

Comment:
The intention with the `tableView` macro was that we would eventually pull it out into something that designers could use in a more manual way, by manually supplying rows, columns and headers. Hence the `hideHeaders` option, whilst not used by `importerTableView` was intended for that. 

Note that in the current implementation, you still have to pass a full array of headers even if you don't want them displayed because the table footer references the header array to do a merged cell. We could remove that and then say "don't pass any headers if you don't want anything displayed".

But also, looks like there was a bug in my work – there is a check on line 15 for `tableHeaders` which should now be `headers`.

Reply:
Not really ready for review yet, but thanks for hilighting the bug :D

Reply:
I wonder if, given that the design system tends to use objects rather than individual parameters (e.g. ``` {{ govukInput({
    label: {
      text: "Address line 1"
    },
    id: "address-line-1",
    name: "addressLine1",
    autocomplete: "address-line1"
  }) }}``` whether we should take the same approach and remodel our parameters as objects.  There's obviously a journey with the other components, but we could start with the tableview and migrate the others over when we choose a final namespace naming scheme?

Reply:
Implemented this, so think it's okay to resolve.